common/protocol_parser/src/lib.rs

```diff
@@ -345,6 +355,14 @@ fn next_field_set(buf: &str) -> Result<Option<(FieldSet, usize)>> {
         if tok_end == 0 {
             tok_end = buf.len()
         }
+        if tok_offsets[0] == tok_offsets[1] {
```

Comment:
what if  filed name has only one char？

Reply:
that tok_offsets[0] == tok_offsets[1] + 1